doc/source/getting_started.rst

```diff
@@ -0,0 +1,190 @@
+Getting started
+===============
+
+Overview
+--------
+
+This package facilitates working with data templates that follow the
+format developed by the `Integrated Assessment Modeling Consortium
+(IAMC) <https://www.iamconsortium.org>`__. It supports validation of
+scenario data and region processing, which consists of renaming and
+aggregation of model “native regions” to “common regions” used in a
+project.
+
+A **DataStructureDefinition** class contains **CodeLists** for
+*variables* (including units) and *regions* to be used in a model
+comparison or scenario exercise following the IAMC data format.
+
+A **CodeList** is a list of “allowed terms” (or codes), where each term
+can have several attributes (e.g., description, unit, parent region).
+
+A **RegionAggregationMapping** is a mapping that defines on a per-model
+basis how model native regions should be renamed and aggregated to
+comparison regions.
+
+A **RegionProcessor** is a class that holds a list of
+RegionAggregationMappings and a DataStructureDefinition. This class is
+used to facilitate region processing for model comparison studies.
+
+The structure of a datastructure definition
+-------------------------------------------
+
+A **DataStructureDefinition** is initialized from a folder with the
+following structure:
+
+Variables
+~~~~~~~~~
+
+The *variable* codelist of the **DataStructureDefinition** will be read
+from all yaml files located in a folder of that name (including any
+sub-folders). They must be formatted as a list of dictionaries mapping
+the variable (key) to its attributes.
+
+.. code:: yaml
+
+   - Some Variable:
+       description: A short description
+       unit: A unit
+       <other attribute>: Some text (optional)
+
+Every variable must have a **unit**, which should be compatible with the
+Python package `iam-units <https://github.com/iamconsortium/units>`__.
+The unit attribute can be empty, i.e., the variable is *dimensionless*.
+
+Tags
+^^^^
```

Comment:
Good question I think it should not be there as its application is in principle not limited to variables but could also be extended to regions. Do I understand that correctly @danielhuppmann?
As regions are also part of a DataStructureDefinition I would say it makes sense to move the regions section also under the DataStructureDefinition.
I'm afraid I don't understand what you mean by:

> to many levels under "The structure of a datastructure definition" with to less headings, I guess.

Reply:
Sorry for being not precise. I meant having, one heading with one subheading with one sub-subheading (heading 1, 2, 3). I just think it might look better to have at least two headings on each level (2, 3). I hope this is better understandable :sweat_smile:

Reply:
Ah right, ok, good point. If we move regions under DataStructureDefinition and move the Tags up one level we should have three subheadings.

Reply:
Tags should work for all types of codelists, but it is only explicitly tested for variables... Suggest to follow a strategy of "assume it works until proven otherwise, then fix and add tests"...

Reply:
Sounds good, I put it at the same level as regions and variables then.